### PR TITLE
Update README badge

### PR DESCRIPTION
**Pull request by mdeggies**
Point to the build job in GHA instead of the deprecated job in CircleCI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-[![CircleCI](https://circleci.com/gh/hashicorp/consul-terraform-sync/tree/main.svg?style=svg&circle-token=a88491ffa8b02149fc483c29c6b8b91ed771f5a5)](https://circleci.com/gh/hashicorp/consul-terraform-sync/tree/main)
+# Consul-Terraform-Sync [![Build Status](https://github.com/hashicorp/consul-terraform-sync/actions/workflows/build.yml/badge.svg)](https://github.com/hashicorp/consul-terraform-sync/actions/workflows/build.yml)
```
Comment (mkam) [inline README.md:1]: We still use CircleCI to run our tests, so could the build status be added in addition to the CircleCI rather than replacing it?

Comment (lornasong) [reply README.md:1]: Hey @mdeggies! Wanted to follow up to get your thoughts on keeping the CircleCi in addition to the GHA badge?

Comment (mdeggies) [reply README.md]: So sorry I missed these github notifications, and thank you for the extra ping to bring this to my attention! I updated this to include both the github action build status, and the circleci tests. It looks like this in a markdown editor online: 

Screen Shot 2022-04-08 at 3 50 40 PM

Comment (mdeggies) [reply README.md]: Let me know if this looks okay or if there are any other changes to make. 👍

**Review by lornasong — APPROVED** [review]
thanks @mdeggies, this looks great!

**Comment by wilkermichael** [comment]
Hi @mdeggies just checking in to see if this is ready to merge?

**Pull request merged**